code/src/components/Survey.js

```diff
@@ -0,0 +1,149 @@
+                    <>
+                        <h2>Housing</h2>
+                        <div className="question-section">
+                            <TextInput 
+                                question="Where do you live?" 
+                                inputAnswer={city} 
+                                inputSet={setCity} 
+                                showNext={() => showNextQuestion('question4', city)} />
+                        </div>
+                    </>
+                    )} 
+                    
+                    {section === 'question4' && (
+                    <>
+                        <h2>Housing</h2>
+                        <div className="question-section">
+                            <SelectInput 
+                                question="Do you live in a house or apartment?" 
+                                setSelectValue={setHousing} 
+                                showNext={() => showNextQuestion('question5', housing)} />
+                        </div>
+                    </>
+                    )}
+
+                    {section === 'question5' && (
+                    <>
+                        <h2>Housing</h2> 
+                        <div className="question-section">   
+                            <RadioInput 
+                                question="I always turn off the light when leaving a room." 
+                                radioValue={lights} 
+                                setRadioValue={setLights}showNext={() => showNextQuestion('question6', lights)} />
+                        </div>
+                    </>
+                    )}
+
+                    {section === 'question6' && (
+                        <div className="question-section">
+                            <h3>Ready to submit?</h3>
+                            <input type="submit" value="Submit" className="button"/> 
+                        </div>
+                    )}
+                </form>
+            </>
+        ); }
+
+    else 
+    //Shows the result page if the form is submitted 
+        return ( 
+            <section className="results" id="results">
+                <h2>Hi {name}</h2>
+                <h3>Thank you for answering our survey. Below you can find a summary of your anwsers.</h3>
+                <div className="answers">
+                        <p>Your concern about climate change: <span className="result-span"> {footprint}/5</span></p>
+                        <p>Your living situation: <span className="result-span"> {housing}</span></p>
+                        <p>Your habits concering lights: <span className="result-span"> {lights}/5</span></p>
+                        <p>Your concern about climate change: <span className="result-span"> {footprint}/5</span></p>
+                </div>
+
+                {/* Shows the first question again and hides the result page */}
+                <button className="button" onClick={showForm}>Back</button>
```

Comment:
When I go back, the radio buttons and text input fields still have the answers applied to them. Not sure if this is the intention or if you want the field cleared for the user to start all over again?